### PR TITLE
Feature/test get offer

### PR DESCRIPTION
**Pull request by Lex-23**
- update flow (views) for get offer and offers
- update tests for get offers
- write tests for get offer

### CODE REVIEW

File: cryptostock/utils/services.py [hunk 1]
```diff
@@ -81,3 +81,11 @@ def offer_flow(offer_count, request, deal) -> dict:
...
     serializer = OfferSerializer(offer)
     return serializer.data
+
+
+def get_offers(request):
+    if request.auth["user_role"] == "client":
+        offers = Offer.objects.filter(client__id=request.user.account.id)
+    elif request.auth["user_role"] == "broker":
+        offers = Offer.objects.filter(deal__broker__id=request.user.account.id)
+    return offers
```
Comment (vadim-denisov) [inline cryptostock/utils/services.py:91]: https://www.django-rest-framework.org/api-guide/filtering/#filtering-against-the-current-user

Comment (vadim-denisov) [reply cryptostock/utils/services.py:91]: `get_queryset` is more suitable for that.

Comment (Lex-23) [reply cryptostock/utils/services.py]: fixed

**Review by vadim-denisov — APPROVED** [review]

**Comment by Lex-23** [comment]
[write pytests](https://trello.com/c/KrUqCZfN/6-write-pytests)

**Pull request merged**